### PR TITLE
MWPW-145191 - Promote Graybox files after MDAST cleanup

### PR DESCRIPTION
**Pull request by arshadparwaiz**
- Uploading files after MDAST processing to the Default folder from Graybox folder
- Promote Copy files without graybox styles from Graybox to Default Folder

### CODE REVIEW

File: actions/graybox/promote-worker.js [hunk 1]
```diff
@@ -173,6 +225,39 @@ async function findAllGrayboxFiles({
     return gbFiles;
 }
 
+/**
+ * Copies the Graybox files back to the main content tree.
+ * Creates intermediate folders if needed.
+ */
+/**
+async function promoteCopy(srcPath, destinationFolder) {
```
Comment (sukamat) [inline actions/graybox/promote-worker.js:233]: Needs clean up?

File: actions/helixUtils.js [hunk 2]
```diff
@@ -148,10 +159,13 @@ class HelixUtils {
                 await this.bulkJobStatus(jobName, operation, repo, bulkPreviewStatus, retryAttempt + 1);
             } else if (response.ok) {
                 const jobStatusJson = await response.json();
-                logger.info(`jobStatusJson ${JSON.stringify(jobStatusJson)}`);
-                logger.info(`${operation} progress ${JSON.stringify(jobStatusJson.progress)}`);
+                // logger.info(`jobStatusJson ${JSON.stringify(jobStatusJson)}`);
+                // logger.info(`${operation} progress ${JSON.stringify(jobStatusJson.progress)}`);
                 jobStatusJson.data?.resources?.forEach((rs) => {
-                    bulkPreviewStatus[rs.path] = { success: JOB_STATUS_CODES.includes(rs.status), resourcePath: rs?.resourcePath, responseCode: rs.status };
+                    logger.info(`RS sourcein here: ${JSON.stringify(rs?.source?.name)}`);
```
Comment (sukamat) [inline actions/helixUtils.js:165]: JSON.stringify is expensive. I would not suggest using this at info level. This is used at few other places in this PR.

File: actions/graybox/promote-worker.js (hunk 1 above)
Comment (arshadparwaiz) [reply actions/graybox/promote-worker.js]: Yeah @sukamat, done now. Thanks!

File: actions/helixUtils.js (hunk 2 above)
Comment (sukamat) [reply actions/helixUtils.js]: I still see a few usages of it
example:
https://github.com/adobecom/graybox-io/pull/11/files#diff-13937d3a10cb9be0c07e932645d50201d8570cf4c57c7db05c0036178057f6ecR156-R159

Comment (arshadparwaiz) [reply actions/helixUtils.js]: Updated these Sunil. Thanks!

**Review by sirivuri — APPROVED** [review]

File: actions/helixUtils.js (hunk 2 above)
Comment (sukamat) [reply actions/helixUtils.js]: @arshadparwaiz Note that you can keep those in if needed. Use `logger.debug(..)` instead. The logger defaults to info logs but you can pass in the log level in `getAioLogger()` before using it.

**Pull request merged**